src/typing_extensions.py

```diff
@@ -1925,6 +1961,51 @@ class Movie(TypedDict):
         """)
 
 
+if hasattr(typing, 'ReadOnly'):
+    ReadOnly = typing.ReadOnly
+elif sys.version_info[:2] >= (3, 9):  # 3.9-3.12
+    @_ExtensionsSpecialForm
+    def ReadOnly(self, parameters):
+        """A special typing construct to mark a key of a TypedDict as read-only.
```

Comment:
```suggestion
        """A special typing construct to mark an item of a TypedDict as read-only.
```
If we want to keep to the naming approach of the PEP